test/e2e/support/utils/save-post-as-draft.js

```diff
@@ -4,7 +4,7 @@
  *
  * @return {Promise} Promise resolving when draft save is complete.
  */
-export async function saveDraft() {
+export async function savePostAsDraft() {
```

Comment:
Are these things always posts though? What about pages, CPTs, etc.? I guess it's always tough and we don't want to be too generic with words like "content" 🤷‍♂️

Reply:
I followed the class name used 🤷‍♂️

Reply:
Pages use the same controls and class names, so I'd prefer to see this as `saveDraft`. I don't see any reason why someone writing a test should have to import different things, or end up mixing terminologies depending on what they're working on, when `saveDraft` would do identical things in both cases and the person writing the test shouldn't need to think about it.

Reply:
@notnownikki makes a good point, let's keep these names generic when they work across different content types.

Reply:
I will revert the name change 👍